apps/CMakeLists.txt

```diff
@@ -48,17 +48,26 @@ function(add_app_test NAME)
         "-DLLVM_DIR=${LLVM_DIR}"
         "-DHalide_TARGET=${Halide_TARGET}")
 
+    # Note that we skip all tests that have the 'benchmark' label; some of the apps
+    # have benchmarks that are in the form of tests, and running them via this path
+    # isn't very useful, since we aren't guaranteed that the machine won't be under
+    # load with other tests. (Also, some of the apps have extensive benchmarks that
+    # take quite a while to run; e.g. apps/linear_algebra, which has over 500.)
+    #
+    # Note also that we specify `--build-noclean` explicitly; apparently the default
+    # for --build-and-test is to 'clean' first, which shouldn't be necessary here.
```

Comment:
What does this mean for things like local laplacian? It just has one binary, which runs a benchmark. In the cmake build it just has the label "internal_app_tests"

Reply:
That test isn't labeled with `benchmark`, so it isn't affected.